### PR TITLE
Labels security equipment as being lethal, nonlethal, less-than-lethal or highly destructive. 

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Labels a few bits of security equipment as lethal, nonlethal, less-than-lethal or highly destructive (in the case of ion carbines or flamethrowers). For shotgun shell boxes, ammo boxes and weapon crafting kits, it clarifies this in the name. For printable individual ammo, it clarifies this in the print name.

## Why It's Good For The Game

[This thread](https://tgstation13.org/phpBB/viewtopic.php?p=676311#p676311) reminded me that I've seen a lot of confusion about various pieces of security equipment and what exactly the distinction is between lethal, nonlethal and less-lethal actually is. People actively use a lot of less-lethal equipment while thinking that it is nonlethal. It isn't. You absolutely can kill someone with rubber shot and beanbags, and the AI will get up in your grill about it.

The same can be said about weapons such as the energy crossbow. I saw one person flabbergasted that by repeatedly shooting someone with it, they killed their prisoner with toxin damage. While the weapon is mostly stamina damage, it still deals a hefty amount of toxin damage, so shouldn't be used necessarily in place of a disabler or baton. Equally, I've seen some people using temperature guns recklessly and finding out far too late that they've murdered a lizardperson by shooting them once, or wondering why the AI is angry at them for using it when it doesn't _seemingly_ cause damage immediately. This has resulted in administrative issues.

We can't assume our players know these distinctions before utilizing this equipment, so having some helpful gear titles will hopefully inform them before they walk into these problems.

## Changelog
:cl:
qol: Clarifies in various names and descriptions whether security equipment is lethal, nonlethal, less-than-lethal or destructive.
/:cl:

### CODE REVIEW

**Comment by GuillaumePrata** [comment]
> ``name = "ion carbine parts kit (Nonlethal/Highly Destructive/Very Lethal (Silicons))"``

It even works as a tutorial feature helping new players know what some tools are designed for, amazing PR.

**Comment by Pinkufeck** [comment]
Godspeed.

**Comment by BlueMemesauce** [comment]
The ion carbine description is the main one that doesn't work. It is way too long, but still doesn't include that it is lethal for people with implants or cybernetic organs.

Also, "Warcrime Lethal" and "Very Lethal" are unnecessary, the amount of lethality is already shown by the bluespace tag.

Finally, "Contraband" for the crossbow is completely pointless, why is that included.

**Comment by necromanceranne** [comment]
> The ion carbine description is the main one that doesn't work. It is way too long, but still doesn't include that it is lethal for people with implants or cybernetic organs.

I could remove the 'nonlethal' descriptor and change it from Silicons to Synthetic. I think, however, it should be teaching people the primary use of the ion carbine, which is combating silicons. I would want that to be emphasized over whether it causes organ failure in people with synthetic organs or stuns someone with cyberlimbs.

> Also, "Warcrime Lethal" and "Very Lethal" are unnecessary, the amount of lethality is already shown by the bluespace tag.

For the latter, it teaches people 'hey, .357 is _particularly_ dangerous' and maybe they'll remember that if they see it later in action. Also it's a joke.

The former is also about hellfire being consistently referred too as a warcrime to use it. It's just a part of that same gag. We don't have to be deathly serious about labels all the time if it isn't necessarily untrue and still part of the joke. Why am I even having to explaining this :U

> Finally, "Contraband" for the crossbow is completely pointless, why is that included.

It's from illegal tech. Labelling it as 'contraband' isn't inaccurate. I think it could maybe cause some drama if people started to think that way about utilizing illegal tech goods.

**Comment by Mothblocks** [comment]
it should be silicon and not synthetic. the point of this is to help new players, not be pedantically correct

File: code/datums/components/crafting/guncrafting.dm [hunk 1]
```diff
@@ -19,44 +19,44 @@
...
-/obj/item/weaponcrafting/gunkit/
+/obj/item/weaponcrafting/gunkit
 	name = "generic gun parts kit"
 	desc = "It's an empty gun parts container! Why do you have this?"
 	icon = 'icons/obj/weapons/improvised.dmi'
 	icon_state = "kitsuitcase"
 
 /obj/item/weaponcrafting/gunkit/nuclear
-	name = "advanced energy gun parts kit"
+	name = "advanced energy gun parts kit (Lethal/Nonlethal)"
```
Comment (Mothblocks) [inline code/datums/components/crafting/guncrafting.dm:29]: the capitalization is a bit ugly dont you think

**Review by Mothblocks — APPROVED** [review]
the ion carbine wording puts me off but i can't think of anything better that's useful

**Comment by BlueMemesauce** [comment]
> it should be silicon and not synthetic. the point of this is to help new players, not be pedantically correct

No, the point is so that people cant make the gun and  shoot it at someone and say "I didnt know it would kill them I didnt read the description!". If a new player wants to know what it does, then they will read the description and bluespace tag.

The ion gun should say that it's lethal for people with cybernetics.

**Comment by necromanceranne** [comment]
Well, in turn, you can't know if someone has cyberware in the first place. It has no external indicators. Even if they did know about how EMPs interact with them, they may not know the person they are shooting has them, as not all implants have a visible effect upon being emp'd.

**Comment by Wallemations** [comment]
I'm not personally a fan of labeling everything in the autolathe as lethal/non-lethal using the `name` var, but as I can't think of a better alternative I guess it'll do

**Comment by BlueMemesauce** [comment]
Also shouldn't ion gun be "harmless" like the foam darts? It doesn't deal any stamina damage.

**Comment by necromanceranne** [comment]
that's not quite accurate given that EMPs are definitely dangerous. I think that's why 'Highly Destructive' is an apt descriptor. Because it is. It breaks electronics. That to me isn't harmless.

**Comment by necromanceranne** [comment]
also I'm conflicting with my own prs aaaah I'll fix it when I wake up

**Comment by BlueMemesauce** [comment]
I agree with the ion gun being labeled highly destructive, but the full thing is currently nonlethal/highly destructive/very lethal (silicons)

Nonlethal is for things that deal stamina damage but no actual damage, which is why riot darts are labeled nonlethal. Harmless is for things that deal no damage, which is why foam darts are labeled harmless. So the ion gun should be labeled harmless to fit with the foam darts.

**Comment by Mothblocks** [comment]
> No, the point is so that people cant make the gun and shoot it at someone and say "I didnt know it would kill them I didnt read the description!". 

There's no catch all term that I would expect every new player to learn and I want to keep it the way it is

@necromanceranne Ping me when conflicts are resolved for merge

**Comment by necromanceranne** [comment]
@Mothblocks all done!

**Comment by SinfulBliss** [comment]
putting (LETHAL) next to every single ammo type just to show that beanbags/rubbers are semi-lethal looks kind of silly. it doesn't really need to be said that 357 ammo, incendiaries, flamethrowers, 38s, etc. are lethal and destructive

**Pull request merged**